From 1e5af85a1b7b4094f2163f09f1ba4435cb1100cc Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Wed, 2 Mar 2016 13:37:29 +0800
Subject: [PATCH] MA-7665 skip build Exchange on the platform without GPU 3D

Exchange apk can't pass build on 6sl and 7d platform, root cause
is Exchange apk needs to link some resource in Email apk, but we
already pushed a patch to skip build Email apk on 6sl and 7d due
to there is no GPU 3D to support OpenGL ES 2.0. To avoid Exchange
build error we need to skip build Exchange also.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 662c52f..cf7a34d 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ifeq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -48,3 +49,4 @@ include $(BUILD_PACKAGE)
 
 # additionally, build unit tests in a separate .apk
 include $(call all-makefiles-under,$(LOCAL_PATH))
+endif
-- 
1.8.0

